<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonDialDialog</class>
 <widget class="QDialog" name="MonDialDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_num_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>120</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_7">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_9">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>280</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_8">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_dot">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>200</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>

退格</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_5">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_0">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>360</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDial">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>

连接</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_shop">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>#</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_6">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>200</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>441</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReturn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>80</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>80</width>
     <height>80</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>80</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>

返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDisplay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>32</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_num_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>120</y>
     <width>100</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1677215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>AlArabiya</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <zorder>frame</zorder>
  <zorder>pushButton_num_2</zorder>
  <zorder>pushButton_num_7</zorder>
  <zorder>pushButton_num_9</zorder>
  <zorder>pushButton_num_8</zorder>
  <zorder>pushButton_dot</zorder>
  <zorder>pushButtonClear</zorder>
  <zorder>pushButton_num_5</zorder>
  <zorder>pushButton_num_4</zorder>
  <zorder>pushButton_num_0</zorder>
  <zorder>pushButton_num_1</zorder>
  <zorder>pushButtonDial</zorder>
  <zorder>pushButton_shop</zorder>
  <zorder>pushButton_num_6</zorder>
  <zorder>pushButtonReturn</zorder>
  <zorder>lineEditDisplay</zorder>
  <zorder>pushButton_num_3</zorder>
 </widget>
 <tabstops>
  <tabstop>pushButton_num_1</tabstop>
  <tabstop>pushButton_num_2</tabstop>
  <tabstop>pushButton_num_3</tabstop>
  <tabstop>pushButton_num_4</tabstop>
  <tabstop>pushButton_num_5</tabstop>
  <tabstop>pushButton_num_6</tabstop>
  <tabstop>pushButton_num_7</tabstop>
  <tabstop>pushButton_num_8</tabstop>
  <tabstop>pushButton_num_9</tabstop>
  <tabstop>pushButton_dot</tabstop>
  <tabstop>pushButton_num_0</tabstop>
  <tabstop>pushButton_shop</tabstop>
  <tabstop>pushButtonDial</tabstop>
  <tabstop>pushButtonClear</tabstop>
  <tabstop>pushButtonReturn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
